pkg/porter/build_test.go

```diff
@@ -59,7 +59,7 @@ func TestPorter_Build(t *testing.T) {
 
 	stamp, err := configadapter.LoadStamp(bun)
 	require.NoError(t, err)
-	assert.Equal(t, "d421a6249dfbdba79e26e866da7533d59590565708dfdb32423cf989f588d0ea", stamp.ManifestDigest)
+	assert.NotEmpty(t, stamp.ManifestDigest)
```

Comment:
I've gotten super tired of checking and updating the exact manifest digest. We just update it every time we tweak the bundle which isn't helpful.